_tutorials/102_march/step-12/102_march_step-12-desc.html

```diff
@@ -0,0 +1,23 @@
+<div>
+    <p id="tutorialGoal">Well done, you'are almost there. In this step, you will add the last missing detail of the example: the slur in the third measure.</p>
+
+    <p>There are two different ways, to encode a slur in MEI:</p>
+    <ol>
+        <li>You can use a <code>@slur</code> attribute on <code>&lt;note&gt;</code> or <code>&lt;chord&gt;</code> with a value of <code>i=initial</code>, <code>m=medial</code> or <code>t=terminal</code>.</li>
+        <li>You can use a separate <code>&lt;slur&gt;</code> element.</li>
+    </ol>
+
+    <p>In this tutorial step, you will use the second approach: Like dynamics, <code>&lt;slur&gt;</code> elements are control events, too, what means, that they are not encoded as child elements of the corresponding events they are meant to control but in most cases outside the <code>staff</code> elements.</p>
```

Comment:
This sentence should be broken into two sentences. "...elements are control events. They are not encoded..."

Reply:
Done